docs/content/docs/deep/summaryTelemetry.md

```diff
@@ -252,40 +239,72 @@ The event cancels in response to a summary nack op for this attempt, an error al
 
 - `message` - message indicating result of summarize attempt; possible values:
```

Comment:
Similar as above, can we use `reason` or `error` (if either is not already used)?

Reply:
Reason is already logged, yes, though in this case it's the reason for kicking off the summary, not the reason that it failed.